openapi/go.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ARGC=$#
+
+if [ $# -ne 2 ]; then
+    echo "Usage:"
+    echo "    go.sh OUTPUT_DIR SETTING_FILE_PATH"
+    echo "    Setting file should define KUBERNETES_BRANCH, CLIENT_VERSION, and PACKAGE_NAME"
+    exit 1
+fi
+
+
+OUTPUT_DIR=$1
+SETTING_FILE=$2
+mkdir -p "${OUTPUT_DIR}"
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
+pushd "${SCRIPT_ROOT}" > /dev/null
+SCRIPT_ROOT=`pwd`
+popd > /dev/null
+
+pushd "${OUTPUT_DIR}" > /dev/null
+OUTPUT_DIR=`pwd`
+popd > /dev/null
+
+source "${SCRIPT_ROOT}/client-generator.sh"
+source "${SETTING_FILE}"
+
+SWAGGER_CODEGEN_COMMIT=v2.2.3; \
```

Comment:
swagger-codegen `v2.3.0` contains some [bug fixes / enhancements for Go client](https://github.com/swagger-api/swagger-codegen/pulls?utf8=%E2%9C%93&q=is%3Apr%20milestone%3Av2.3.0%20is%3Amerged%20%20label%3A%22Client%3A%20Go%22%20) 

We could use `v2.3.1` which is the current stable. Since we have not generated the client yet, we are not introducing breaking change to our users.